recipes/hello_tests/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# A temporary hack.
+source activate _build
```

Comment:
We have to do this until ( https://github.com/conda/conda-build/issues/910 ) is addressed somehow. Though this is still better than copying loads of environment variables around.